lib/manageiq/rpm_build/generate_gemset.rb

```diff
@@ -138,6 +138,9 @@ def generate_npm_manifest
       def run_license_finder(repos, type)
         where_am_i
 
+        shell_cmd("license_finder ignored_groups add development")
+        shell_cmd("license_finder ignored_groups add test")
```

Comment:
Dumb question.  Does this work? I don't see other calls to `license_finder` with `ignored_groups`.  Gemfile.lock manipulation is tricky so want to make sure that works as we want.

Reply:
I guess the change that we're trying to workaround will break systems with the rpms installed if you run bundle as it will fully resolve the Gemfile and look for the "ignored" dependencies in order to resolve the dependency tree even if it doesn't need to install/use them. 

I don't know if that is a new problem.

Reply:
I ran a build locally and it works.  Build succeeds, manifest is generated, dev dependencies are not in the resulting RPMs.

Reply:
I don't think running `bundle install` on a RPM system was ever supported.  I think it will work because we ship the Gemfile.lock and the bundler config in the RPMs, so it should be a no-op.